privacy-snapshot.json

```diff
@@ -90,11 +90,13 @@
   "raw.githubusercontent.com",
   "registry.npmjs.org",
   "responsive-rpc.test",
+  "ruleset-engine.dev-api.cx.metamask.io",
```

Comment:
We are making network requests to dev ruleset-engine api for the shield transaction tests.

cc: @MetaMask/extension-privacy-reviewers